modules/services/src/main/scala/cards/nine/services/free/interpreter/collection/Services.scala

```diff
@@ -47,62 +47,60 @@ class Services(
       )
     }
 
-  def getById(id: Long): PersistenceService[SharedCollection] =
+  override def getById(id: Long): PersistenceService[SharedCollection] =
     collectionPersistence.fetchOption(Queries.getById, id) map (
       Either.fromOption(_, SharedCollectionNotFound("Shared collection not found"))
     )
 
-  def getByPublicIdentifier(publicIdentifier: String): PersistenceService[SharedCollection] =
+  override def getByPublicId(publicIdentifier: String): PersistenceService[SharedCollection] =
     collectionPersistence.fetchOption(
```

Comment:
I'm not sure if we the doobie integration in Freestyle we could get rid of the Persistence Service.